packages/template/src/views/Data.vue

```diff
@@ -2,6 +2,7 @@
   <div class="page-background">
     <SpCrud
       :storeName="'cosmonaut.blog.blog'"
+      :itemName="'Post'"
```

Comment:
If the property type were declared as String, I think you would make it easier to call here with just `storeName="cosmonaut.blog.blog"` and `itemName="Post"`